src/ts-node.ts

```diff
@@ -1,7 +1,15 @@
 import * as fs from 'fs'
 import * as path from 'path'
 import * as TSNode from 'ts-node'
-import {parseConfigFileTextToJson} from 'typescript'
+
+let parseConfigFileTextToJson
+try {
+  // Lazy-require 'typescript' to support JavaScript @oclif projects.
+  ({parseConfigFileTextToJson} = require('typescript'))
```

Comment:
This should probably just be done with dynamic import behaviors, something like the following:

```ts
async function loadTSConfig(root: string): TSConfig | undefined {
  const tsconfigPath = path.join(root, 'tsconfig.json')
  if (fs.existsSync(tsconfigPath)) {
    const { parseConfigFileTextToJson } = await import('typescript')
```